package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "shop",
+  "private": true,
+  "scripts": {
+    "start": "prpl-server --root build/ --host 0.0.0.0 --https-redirect --bot-proxy https://render-tron.appspot.com/render"
```

Comment:
Right now I suppose you deploy all the repository content to GAE.
I think it would be better to upload only the `build` folder.

Reply:
I have tried with this script:

```javascript
const fs = require('fs');
const path = require('path');

const buildPath = require('polymer-cli/lib/build/build').mainBuildDirectoryName;

function copyFile(fileName) {
  const sourceFile = path.join(__dirname, fileName);
  const destinationFile = path.join(buildPath, fileName);

  fs.copyFileSync(sourceFile, destinationFile);
}

copyFile('package.json');
copyFile('app.yml');
```

And then:

```
gcloud app deploy build/
```